status/bioinfo_analysis.py

```diff
@@ -74,7 +74,7 @@ def post(self, project_id):
         neg_save_res = []
         for res in save_result:
             if not res[0]:
-                neg_save_res.append[res[1]]
+                neg_save_res.append(res[1])
```

Comment:
Ah, but I get the `Not all changes saved in bulk update! Doc ids that failed to update are shown below` -error still 🤔 but maybe that's due to the connection error?

Reply:
Hm it's just my user, maybe you need to be an admin?

Reply:
You could use the genstat general user, it should have the required permissions.